src/apps/Console/ConsoleFeedback.js

```diff
@@ -216,7 +216,38 @@ export default function ConsoleFeedback({
         </div>
```

Comment:
Building on https://github.com/aragon/aragon/pull/1374, I wonder if we should also add a message at the end of any feedback we give about command arguments, pointing to the relevant documentation, e.g.

> "For more information, see the [documentation](https://github.com/aragon/aragon/blob/master/docs/CONSOLE.md#exec).

Reply:
Hmm, I feel this would cause too much clutter on the feedback; it might be better to put a link to the documentation at the start when we only show the commands available.